libtock/unit_test.c

```diff
@@ -230,7 +231,7 @@ void unit_test_runner(unit_test_fun *tests, uint32_t test_count,
 
   // Establish communication with the test supervisor service. First delay 10 ms
   // to ensure the supervisor service has time to register.
-  delay_ms(10);
+  //delay_ms(10); // TODO: Restore me once unit test is ported and moved
```

Comment:
Fix TODO.

Reply:
That is a TODO intended to be resolved outside of this PR -- this PR targets the libtock-folders branch, and the PR for that branch (https://github.com/tock/libtock-c/pull/370) tracks the remaining drivers that need to be ported, including this one.